src/firebaseConfig.ts

```diff
@@ -102,13 +102,14 @@ export type FirestoreConfig = {
   indexes?: string;
 } & Deployable;
 
-export type FunctionsConfig = {
-  // TODO: Add types for "backend"
+export type FunctionConfig = {
   source?: string;
   ignore?: string[];
   runtime?: CloudFunctionRuntimes;
 } & Deployable;
 
+export type FunctionsConfig = FunctionConfig | FunctionConfig[];
```

Comment:
Function*s* vs Function config is incredibly subtle. I might prefer names as clear as FunctionsSingle | FunctionsMultiple